target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-3720-gl-mv1000.dts

```diff
@@ -29,6 +29,28 @@
 			3300000 0x0>;
 		enable-active-high;
 	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led-1 {
+			label = "gl-mv1000:green:vpn";
```

Comment:
If you add this, please use the recently preferred syntax without the useless model, i.e. "green:vpn" etc.